### PR TITLE
Bunny hop walking animation

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request

Source Inspiration: https://github.com/tgstation/tgstation/pull/79698
This PR makes it so that you can take an optional quirk that makes it so you bounce like a rabbit.
This PR doesn't add anything to Genetics, or fix those bugs, it simply makes you bounce. :)

## Why It's Good For The Game

This will allow bunny or other characters to actually hop! Adds a little bit of personal expression.

## Proof Of Testing
https://github.com/Bubberstation/Bubberstation/assets/110273561/5ec80d47-316f-495e-b064-9c94b725b36a

## Changelog

:cl: ReturnToZender (port), IAmGoofBall (original code)
add: Bunny hop quirk for all players to choose! :) Rabbit simplemobs automatically do this hop.
/:cl:

### CODE REVIEW

**Comment by Majkl-J** [comment]
Can we make it maybe a custom option like with some quirks to avoid the bloat of hopping/waddle?

**Comment by ReturnToZender** [comment]
> Can we make it maybe a custom option like with some quirks to avoid the bloat of hopping/waddle?

You mean like a character preference?

**Comment by Majkl-J** [comment]
> > Can we make it maybe a custom option like with some quirks to avoid the bloat of hopping/waddle?
> 
> You mean like a character preference?

No, certain quirks can have extra settings that can be modified... lemme grab an example in a bit

**Comment by nikothedude** [comment]
> > Can we make it maybe a custom option like with some quirks to avoid the bloat of hopping/waddle?
> 
> You mean like a character preference?

they mean, bundle this with waddling via quirk customization

**Comment by Majkl-J** [comment]
> > > Can we make it maybe a custom option like with some quirks to avoid the bloat of hopping/waddle?
> > 
> > 
> > You mean like a character preference?
> 
> they mean, bundle this with waddling via quirk customization

Thank you I am having a stroke today

File: tgstation.dme [hunk 1]
```diff
@@ -8606,6 +8606,7 @@
 #include "modular_zubbers\modules\quirks\code\negative_quirks\sensitive_snout.dm"
 #include "modular_zubbers\modules\quirks\code\neutral_quirks\hungry.dm"
 #include "modular_zubbers\modules\quirks\code\neutral_quirks\waddle.dm"
```
Comment (nevimer) [inline tgstation.dme:8608]:
```suggestion

```

**Review by nevimer — CHANGES_REQUESTED** [review]

**Comment by Swiftfeather** [comment]
This has been inactive for over a month. Reopen if you come back to it.

**Pull request closed without merging**